packages/manager/modules/octavia-load-balancer/src/translations/Messages_fr_FR.json

```diff
@@ -1,4 +1,20 @@
 {
   "octavia_load_balancer": "Load Balancer",
-  "octavia_load_balancer_global_error": "L'erreur suivante est survenue : <br> {{message}}<br> request_id: {{requestId}}"
+  "octavia_load_balancers": "Load Balancers",
+  "octavia_load_balancer_global_error": "L'erreur suivante est survenue : <br> {{message}}<br> request_id: {{requestId}}",
```

Comment:
Avoid using html in translation files

Reply:
This translation was already there and done by Quentin, not sure if I can change it.